examples/preprocessing/plot_artifact_correction_with_faster.py

```diff
@@ -0,0 +1,89 @@
+"""
+====================================
+EEG artifact correction using FASTER
+====================================
+
+In this example, a variety of metrics are use to detect channels, epochs and
+ICA components that contain artifacts. Rejection and interpolation are used to
+clean the EEG data.
+
+References
+----------
+[1] Nolan H., Whelan R. and Reilly RB. FASTER: fully automated
+    statistical thresholding for EEG artifact rejection. Journal of
+    Neuroscience Methods, vol. 192, issue 1, pp. 152-162, 2010.
+"""
+import mne
+from mne import io
+from mne.preprocessing import (find_bad_channels, find_bad_epochs,
+                               find_bad_channels_in_epochs, ICA)
+from mne.datasets import sample
+
+# Load raw data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+
+raw = io.Raw(raw_fname, preload=True)
```

Comment:
I think I remember @agramfort saying it's better to use `io.read_raw_fif` as it is clearer from the name what it does (for a newcomer).